pyomo/core/tests/data/test_odbc_ini.py

```diff
@@ -15,8 +15,6 @@
 
 import pyutilib.th as unittest
 
-from pyomo.environ import *
```

Comment:
I tried to run these locally and all of them were skipped because "PyODBC is not installed." (but... it *is* installed - I checked. Can you try running that locally and see what happens?)

Reply:
@jsiirola - This test is skipped in `master`, too.

Reply:
There was a bug in this. `pyomo.core.plugins.data.db_table` doesn't exist. It was moved to `pyomo.dataportal.plugins.db_table` at some point. This has been corrected, and I have confirmed that tests run locally (without the `from pyomo.environ import *` statement - even better)